tests_e2e/conftest.py

```diff
@@ -27,6 +27,10 @@
 
 TEST_DIR = Path(__file__).parent
 
+pytest_plugins = [
```

Comment:
Added tests_common as pytest plugin to e2e tests to make possible to run e2e tests locally.
It was already added in integration and unit tests.